File: 백준/Bronze/16968. 차량 번호판 1/차량 번호판 1.py
input = input()
dLen = 10
cLen = 26
check = 0
result = 1
for i in range(len(input)) :
    if i > 0 and input[i - 1] == input[i] :
        check = 1;
    else :
        check = 0;
    if input[i] == "d" :
        if check == 1 : 
            result *= dLen - 1
        else :
            result *= dLen
    else :
        if check == 1 : 
            result *= cLen - 1
        else :
            result *= cLen
print(result)

File: 백준/Bronze/16968. 차량 번호판 1/차량 번호판 1.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim();
const dLen = 10;
const cLen = 26;
let count = 0;
let result = 1;
for(let i = 0; i < input.length; i++){
    if(i > 0 && input[i - 1] === input[i]){
        count = 1;
    } else{
        count = 0;
    }
    if(input[i] === "d"){   
        if(count === 1) result *= dLen - 1;
        else result *= dLen;
    } else{
        if(count === 1) result *= cLen - 1;
        else result *= cLen;
    }
}
console.log(result);